src/logging/timer.h

```diff
@@ -60,8 +60,8 @@ class Timer
 
     std::string LogMsg(const std::string& msg)
     {
-        const auto end_time = GetTime<std::chrono::microseconds>() - m_start_t;
-        if (m_start_t.count() <= 0) {
+        const auto end_time{std::chrono::duration_cast<std::chrono::microseconds>(std::chrono::steady_clock::now() - m_start_t)};
+        if (m_start_t == decltype(m_start_t){}) {
```

Comment:
Have you considered making `m_start_t` a `std::optional`?

Reply:
Thanks, done